### PR TITLE
English corrections, typos and spelling corrections.

### PR DESCRIPTION
**Pull request by Vasanth-Indix**
@paambaati , @codingnirvana,  @manojlds, @praveenselvam, @ashwanthkumar: Went through the content for UI radar and updated with english statement corrections and typo and spelling corrections.
Feel free to run through the content and update this PR for any missed out updates.

### CODE REVIEW

File: source/_data/ui.yml [hunk 1]
```diff
@@ -39,25 +39,25 @@
   ring: Adopt
   isNew: true
   description: "[Sass](http://sass-lang.com/) is an extension of CSS and defines a clean structured way of writing CSS by allowing to modularize CSS snippets while offering a whole slew of features which are practised in javascript environment. Features like variables, external imports, functions and extensibility allow to experience the benefits of modular patterns."
-  opinion: Given its advantages, we actively adopt SASS for styling its web apps. SASS added defined structure to app styling code base and made it easily readable. The extensible nature allowed us to define low level themes and reuse it across apps.
+  opinion: Given SASS advantages, we actively adopt SASS for styling our web apps. SASS adds defined structure to app styling code base and make the code easily readable. The extensible nature allowed us to define low level themes and reuse it across apps.
```
Comment (paambaati) [inline source/_data/ui.yml]: @Vasanth-Indix Do we really need two 'SASS'es in one sentence?

File: source/_data/ui.yml [hunk 2]
```diff
@@ -207,25 +207,25 @@
...
   description: "[Socket.IO](https://socket.io/) is a library that provides comprehensive support for WebSockets."
-  opinion: It is isomorphic (meaning we can use it on both the client & the server sides) and supports automatic reconnection and disconnection detection, making for a very robust and reliable WebSocket library. We use Socket.IO for realtime IPCs between many of our microservices.
+  opinion: It is isomorphic (meaning we can use it on both the client & the server sides) and supports automatic reconnection and disconnection detection, making for a very robust and reliable WebSocket library. We use Socket.IO for real-time IPCs between many of our microservices.
 - name: NPM
   quadrant: Testing & Build Tools
   ring: Adopt
   isNew: true
   description: "[NPM](https://www.npmjs.com/) is the default package manager that ships with Node.js."
-  opinion: For a number of projects, we simply use npm scripts for test runs & builds.
+  opinion: For a number of projects, we simply use npm scripts for test runs & build processes.
```
Comment (paambaati) [inline source/_data/ui.yml:216]: @Vasanth-Indix Isn't _build_ itself succinct enough?

File: source/_data/ui.yml [hunk 3]
```diff
@@ -267,7 +267,7 @@
   ring: Hold
   isNew: true
   description: "[Jasmine](https://jasmine.github.io/) was one of the earliest Javascript testing tools and comes as a package with in-built assertion utilities."
-  opinion: Jasmine doesn't come with a test runner, while Mocha does. We've put Jasmine on hold, with Mocha + Chai being adopted.
+  opinion: Jasmine doesn't come with a test runner, while Mocha does. We've put Jasmine on hold, with Mocha + Chai being at adopt.
```
Comment (paambaati) [inline source/_data/ui.yml]: @Vasanth-Indix Not sure if we need this change; the original flows better even as a standalone sentence.

**Review by paambaati — COMMENTED** [review]
Adding just a few minor nits. Otherwise the changes look good, thanks!

File: source/_data/ui.yml (hunk 1 above)
Comment (Vasanth-Indix) [reply source/_data/ui.yml]: Will change it.

File: source/_data/ui.yml (hunk 2 above)
Comment (Vasanth-Indix) [reply source/_data/ui.yml:216]: The statement sounded to end abruptly. `test runs` is two-worded and `build processes` complement with that.

File: source/_data/ui.yml (hunk 3 above)
Comment (Vasanth-Indix) [reply source/_data/ui.yml]: Ok. On re-reading sounds right. Will change it back.

**Pull request merged**